pkgs/development/libraries/gperftools/default.nix

```diff
@@ -51,6 +51,26 @@ stdenv.mkDerivation rec {
 
   enableParallelBuilding = true;
 
+  # split output to minimize dependencies on dynamic linking.
+  # tcmalloc_minimal goes into a dedicated output to further minimize
+  # the runtime dependencies of applications that just use tcmalloc
+  # as an allocator, without the profiling.
+  outputs = [ "out" "dev" "tcmalloc_minimal" "lib" "static" ];
+  postInstall = ''
+mkdir -p $tcmalloc_minimal/lib
+mkdir -p $static/lib
+mv $lib/lib/*.a $static/lib/
+mv $lib/lib/libtcmalloc_minimal.* $tcmalloc_minimal/lib/
+
+# Ensure that metadata files know about new locations
+substituteInPlace $dev/lib/pkgconfig/libtcmalloc_minimal.pc --replace-fail $lib $tcmalloc_minimal
+substituteInPlace $tcmalloc_minimal/lib/libtcmalloc_minimal.la --replace-fail $lib $tcmalloc_minimal
```

Comment:
Afaiu `libtool` helpers should also go into `dev`, since they are somewhat of a precursor to `pkg-config` files.

Reply:
I looked around for this, and found other split packages where this was not the case, and none that put them in dev. See for example fontconfig. I agree that it seems like it should be a dev thing but I think I should be consistent with other split packages.

Reply:
incidentally, .la files are not precursors to pkg-config files. They are completely separate systems.
In particular, gperftools just constructs the .pc files with a bit of shell scripting. There's no tooling involved there, libtool or otherwise.

Reply:
From `pkgs/build-support/setup-hooks/multiple-output.sh`:
> # don't move *.la, as libtool needs them in the directory of the library

So .la files have to go with the library they describe.

Reply:
> incidentally, .la files are not precursors to pkg-config files. They are completely separate systems.

By "precursor" I meant they partially address the same problems. `la` files where originally designed to be used by other tools, too. But no other tool ever started doing so.
I doubt any direct downstreams of `gperftools` uses libtool in their build setup anyways, so the file is probably irrelevant, and we should not invest too much time into it.